.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,62 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: ex. email@example.com
+    validations:
+      required: false
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
```

Comment:
This might be better as a separate section. What happened, and optionally, what did you expect to happen? This will keep things a little more organized, and will let people report even if there isn't an expectation (or it's obvious, like the site crashes and the expectation is that it doesn't crash).

Reply:
attributes:
      label: If you expected something else to occur what did you expect?
      description: Also tell us, what did you expect to happen?
      placeholder: What did you expect to happen?
    validations:
      required: false

?

Reply:
Hmm. I feel like for a bug report, we need to know what the bug is / what happened, what steps were taken to trigger the bug (this can probably be combined with the previous one), and what they expected to happen (optional).

Reply:
right that was just an excerpt. I left everything else. I'll go ahead and also add a "step to replicate" textarea as well. For brevity here is the Labels for all the sections: What happened, If you expected something else to occur what did you expect?, What operating system are you using?, What browsers are you seeing the problem on?, Relevant log output, Code of Conduct

Reply:
Sounds good!